resource/odvd/opendlvdata.odvd

```diff
@@ -117,6 +117,32 @@ message opendlv.sensation.Voice [id = 166] {
   //list<uint8> data [id = 4];
 }
 
+// according to the GCDC document we have to send the following information from localization:
+//    A.1.12.1 Latitude reference position
+//    A.1.12.2 Longitude reference position
+//    A.1.12.3 Position confidence ellipse interval 95%
+//    A.1.13.1 HeadingValue
+//    A.1.13.2 Heading confidence interval 95%
+//    A.1.14.1 SpeedValue
+//    A.1.14.2 Speed confidence interval 95%
+//    A.1.15.1 YawRateValue
+//    A.1.15.2 Yaw rate confidence interval 95%
+//    A.1.16.1 LongitudinalAccelerationValue
+//    A.1.16.2 longitudinalAccelerationConfidence
+message opendlv.sensation.Geolocation [id = 169] {
+  double latitude [id = 1];
```

Comment:
This message should not be defined in this way. It should include only the global state (position and heading). The local state should be put in another message. I do not know how to treat the confidence values yet, I will think about it for a while...